api/MutationRecord.json

```diff
@@ -29,10 +29,10 @@
             "version_added": true
           },
           "safari": {
-            "version_added": true
+            "version_added": "7"
```

Comment:
Going by https://developer.mozilla.org/en-US/docs/Web/API/MutationObserver#Browser_compatibility it seems more likely this was introduced in Safari 6, but perhaps the interface wasn't exposed then?

Reply:
It looks like `MutationObserver` was supported in Safari 6 behind the `WebKit` prefix, so it's probably the same case here.  Digging deeper into this.

Reply:
Hmm...nope, not a prefix.  It _is_ supported in Safari 6.1, though.  I wonder why "6.2" keeps getting translated into "7"...

Reply:
BCD doesn't have 6.2 as a version. The version matching logic should map it to 6.1, but maybe there's a bug?